frontend/public/components/secrets/create-secret.tsx

```diff
@@ -313,10 +313,14 @@ class ConfigEntryForm extends React.Component<ConfigEntryFormProps, ConfigEntryF
       : this.state.auth;
   }
   changeData(event) {
+    event.persist();
```

Comment:
Instead of this, can we use

```js
const { name, value } = event.target;
```

and reference `name` below instead of `event`?

Reply:
yeah, probably thats for the best :+1: